### PR TITLE
Form view compatible with SQL cases

### PR DESCRIPTION
**Pull request by snopoke**
Doing some QA and found it pretty irritating not being able to easily look at a form... so now you can.
@emord 

@dimagi/scale-team FYI

### CODE REVIEW

**Comment by NoahCarnahan** [comment]
Looks good to me :+1:

**Comment by czue** [comment]
very cool! chipping away... :hammer:

File: corehq/form_processor/admin.py [hunk 1]
```diff
@@ -3,5 +3,11 @@
 
 
 # note: these require ALLOW_FORM_PROCESSING_QUERIES = True in your localsettings.py to work
-admin.site.register(XFormInstanceSQL)
+@admin.register(XFormInstanceSQL)
```
Comment (benrudolph) [inline corehq/form_processor/admin.py]: this is cool that it works even though we don't use the django ORM for it

**Comment by benrudolph** [comment]
nice!

File: corehq/form_processor/admin.py (hunk 1 above)
Comment (snopoke) [inline corehq/form_processor/admin.py]: This only works if we aren't sharding because it does use the Django ORM

**Pull request merged**